hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+Adds a recipe to the docstring of :func:`from_type <hypothesis.strategies.from_type>`
```

Comment:
:func:`~hypothesis.strategies.from_type`

The `~` means that this renders as `from_type()` - we only need the explicit link text version when showing an argument to the function, or to add the `@` to decorators.

I'd also include a sentence motivating recipe in the changelog, e.g. "This is useful when writing tests which check that invalid input is rejected in a certain way."